content/course-specific-material/introR-NatGW-Aug16.md

```diff
@@ -1,11 +1,15 @@
 ---
+author: 
 date: 2016-08-29
 slug: introR-NatGW-Aug16
+draft: True
 title: National GW Workshop
-menu: 
-  main:
-    parent: Course Specific Material
-    weight: 1
+categories: Data Science
+ 
+tags: 
+  - R
+ 
+ 
```

Comment:
@lindsaycarr do the changes above this line look right to you? i didn't edit anything in this part of the Rmd. maybe this got autopopulated with the USGSmarkdowntemplates::hugo?